scripts/docker/enable-winrm.ps1

```diff
@@ -15,7 +15,11 @@ if (Test-Path A:\install-containers-feature.ps1) {
 Stop-Service winrm
 . sc.exe config winrm start= delayed-auto
```

Comment:
Oddly, WinRM never comes back up after the reboot, so vagrant is unable to continue the provisioning steps.